### PR TITLE
asciitopgm: add page

### PR DESCRIPTION
**Pull request by Gauravpadam**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Umbrella issue #10887

This PR adds asciitopgm page documentation

### CODE REVIEW

File: pages/common/asciitopgm.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# asciitopgm
+
+> Convert ASCII graphics into a PGM.
```
Comment (gutjuri) [inline pages/common/asciitopgm.md:3]:
```suggestion
> Convert ASCII graphics into a PGM file.
```

File: pages/common/asciitopgm.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/asciitopgm.html>.
+
+- Read ASCII data as input and produce a PGM image with pixel values that are an approximation of the "brightness" of the ASCII characters:
+
+`asciitopgm inputfile > output.pgm`
```
Comment (gutjuri) [inline pages/common/asciitopgm.md:8]:
```suggestion
`asciitopgm {{path/to/file}} > output.pgm`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thank you. I added some suggestions.

File: pages/common/asciitopgm.md (hunk 1 above)
Comment (Gauravpadam) [reply pages/common/asciitopgm.md]: Sure, I'll do that

**Comment by Gauravpadam** [comment]
@gutjuri I've implemented the suggestions, Would you please take a look?

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**